### PR TITLE
Rendering: Skybox

### PR DESCRIPTION
**Pull request by sp0lsh**

### CODE REVIEW

File: PolyEngine/RenderingDevice/OpenGL/Src/GLRenderingDevice.hpp [hunk 1]
```diff
@@ -103,6 +106,7 @@ namespace Poly
 		EnumArray<std::unique_ptr<RenderingPassBase>, ePostprocessRenderPassType> PostprocessRenderingPasses;
 
 		std::unique_ptr<PostprocessQuad> PostprocessRenderingQuad;
+		std::unique_ptr<PrimitiveCube> PrimitiveRenderingCube;
```
Comment (MuniuDev) [inline PolyEngine/RenderingDevice/OpenGL/Src/GLRenderingDevice.hpp]: Maybe we could create cube with reversed UV to address fliped textures problem

Comment (sp0lsh) [reply PolyEngine/RenderingDevice/OpenGL/Src/GLRenderingDevice.hpp]: Better fix the vertex order in triangles.

**Comment by sp0lsh** [comment]
Cubemaps are difficult to create and iterate. I have prototype of cubemap to lat-long mapping. Will post in another PR.

File: PolyEngine/RenderingDevice/OpenGL/Src/GLRenderingDevice.hpp (hunk 1 above)
Comment (sp0lsh) [reply PolyEngine/RenderingDevice/OpenGL/Src/GLRenderingDevice.hpp]: Vertex order used but it was not very effective.
Scaled UV by vec3(-1.0, 1.0, 1.0) and it was supper effective.
*Skybox flipped*

**Pull request merged**